### PR TITLE
added tabs to page

### PR DESCRIPTION
**Pull request by mordrax**
Initial implementation of tabs, should eventually belong on a page instead of changing the currentPage.

### CODE REVIEW

File: Eventful/View.elm [hunk 1]
```diff
@@ -28,7 +30,25 @@ type alias Mdl =
...
+            , Options.span [ css "width" "4px" ] []
+            , text "Index"
+            ]
+        , Tabs.label [ Options.center ]
+            [ Icon.i "code"
+            , Options.span [ css "width" "4px" ] []
+            , text "Settings"
+            ]
+        ]
+        [ viewHelper model |> Material.Scheme.top ]
```
Comment (JulianLeviston) [inline Eventful/View.elm]: It'd be sweet if the name `viewHelper` was named something more meaningful, like say `renderPageContent` or something.

**Comment by JulianLeviston** [comment]
This looks good to me. 🚢 when you're happy and you've maybe adjusted that one name if you think it's a good idea.

**Comment by mordrax** [comment]
Yeah, `helper` is a way overused vague concept. `render` and `view` are the same, and `content` is extra info. I renamed to `viewPage`

**Comment by mordrax** [comment]
I actually don't like this PR very much because it confused page and tab in the same model. I'd like a 'page' to contain some 'tabs'. This way, tabs has just become pages...

**Pull request merged**